sop-extras/src/Data/SOP/Index.hs

```diff
@@ -18,6 +20,8 @@ module Data.SOP.Index (
   , indices
   , injectNS
   , injectNS'
+  , pattern IS
+  , pattern IZ
```

Comment:
[Bundle](https://downloads.haskell.org/ghc/latest/docs/users_guide/exts/pattern_synonyms.html#import-and-export-of-pattern-synonyms) these?